src/modules/Crop/info.json

```diff
@@ -16,12 +16,12 @@
     "w": {
       "type": "string",
       "desc": "Width of crop",
-      "default": "50%"
+      "default": "100%"
```

Comment:
Why have you changed the defaults? `100%` is no crop. We would want to have at least some crop right?

Reply:
By default it doesn't crop because we are already providing facilty of drag to crop in UI so cropping the image by default doesn't makes sense. If we change this to 50% then in UI, drag to crop is available for that 50% of the image only.

Reply:
Oh!